crates/bevy_render/src/view/mod.rs

```diff
@@ -40,9 +40,21 @@ impl Plugin for ViewPlugin {
 }
 
 #[derive(Clone)]
+/// Configuration resource for [Multi-Sample Anti-Aliasing](https://en.wikipedia.org/wiki/Multisample_anti-aliasing).
+///
+/// # Example
+/// ```
+/// # use bevy_render::prelude::Msaa;
```

Comment:
```suggestion
/// # use bevy_render::prelude::Msaa;
/// # use bevy_app::App;
```

Missing this import in the example